tests/stress-tests.lisp

```diff
@@ -28,9 +28,10 @@
   (write-char #\]) (finish-output)
   nil)
 
-#+unix
 (deftest test-posix-shared-memory-allocation/deallocation ()
   "Test allocation and deallocation of large blocks of POSIX shared memory. This test may result in total system failure."
+  #+windows (skip)
+  #-windows
```

Comment:
Is this to mute warnings on `#+windows` machines?

Reply:
yes that's the idea. Keep the test in name, but no in effect.